src/runtime/value.rs

```diff
@@ -40,11 +63,16 @@ impl Value {
 impl fmt::Display for Value {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
-            &Value::Int(value) => write!(f, "{}", value),
-            &Value::Float(value) => write!(f, "{}", value),
-            &Value::String(ref value) => write!(f, "{}", value),
-            &Value::DivertTarget(ref value) => write!(f, "DivertTarget({})", value.to_string()),
-            &Value::VariablePointer(ref name, _) => write!(f, "VarPtr({})", name),
+            Value::Int(value) => write!(f, "{}", value),
+            Value::Float(value) => write!(f, "{}", value),
+            Value::String(value) => write!(f, "{}", value),
+            Value::DivertTarget { target_path } => {
+                write!(f, "DivertTarget({})", target_path.to_string())
```

Comment:
`target_path` implements display, so we should be able to remove the `.to_string()`